features/step_definitions/rc.rb

```diff
@@ -10,10 +10,10 @@
   raise "replication controller #{rc.name} never became ready" unless @result[:success]
 end
 
-# Given /^I wait until the status of replication controller "(.+)" becomes :(.+)$/ do |resource_name, status|
-#   ready_timeout = 10 * 60
-#   rc(resource_name).wait_till_status(status.to_sym, user, ready_timeout)
-# end
+Given /^I wait until the status of replication controller "(.+)" becomes :(.+)$/ do |resource_name, status|
+  ready_timeout = 10 * 60
+  rc(resource_name).wait_till_status(status.to_sym, user, ready_timeout)
+end
 
```

Comment:
@pruan-rht  do you know why we commented the step about rc?

Reply:
@SSshahan I don't know...git blame seems to indicate it's was done by @akostadinov.

Reply:
@akostadinov  do you know any detail about why we comment them out?

Reply:
one web case need to use them

Reply:
Maybe we better write a generic step `I wait until status of #{WORD} "#{QUOTED}" becomes #{SYM}`, where first parameter is resource type. In this case with one step we can cover all resource types instead of having a separate step for all resources that need.

You can see in `features/step_definitions/resources.rb` how to get resource object from word and name.